### ISSUE TITLE
Issues with long user names

### ISSUE THREAD

**Issue by WC3D** [trustcrypto/OnlyKey-Firmware#156]
Hello, we use 2FA software Authlite. And we discovered with help of their support that the firmware ignores calls to generate shorter Yubico-style codes. Is there a patch in the works to generate shorter codes? On Yubikeys deployed in our environment, we are able to change the values from 16-4 and generate the shortcodes under 64 chars.

**Comment by onlykey** [comment]
Yes, please use the latest firmware here for shorter Yubico OTPs - https://github.com/trustcrypto/OnlyKey-Firmware/releases/tag/v3.0.2-prod

**Comment by WC3D** [comment]
This issue still persists. After the latest update 3.0.3.

**Comment by WC3D** [comment]
Any update

**Comment by onlykey** [comment]
@WC3D Here is new firmware that supports the shorter OTPs and the latest Authlite app 
[Signed_OnlyKey_3_0_4_STD.txt](https://github.com/trustcrypto/OnlyKey-Firmware/files/10187498/Signed_OnlyKey_3_0_4_STD.txt)
